go.mod

```diff
@@ -663,7 +675,10 @@ replace (
 // Required for github.com/hashicorp/vault@v1.17.2
 replace github.com/pires/go-proxyproto v1.0.0 => github.com/peteski22/go-proxyproto v1.0.0
 
-// Required for github.com/docker/docker v26 until https://github.com/hashicorp/go-secure-stdlib/pull/126 is merged
-replace github.com/hashicorp/go-secure-stdlib/plugincontainer v0.3.0 => github.com/odvarkadaniel/go-secure-stdlib/plugincontainer v0.0.0-20240426094400-4af81bf367be
-
 replace github.com/google/cadvisor => github.com/google/cadvisor v0.49.1-0.20240628164550-89f779d86055
+
+// Required for github.com/hashicorp/go-secure-stdlib/plugincontainer@v0.3.1
+replace (
+	github.com/docker/docker => github.com/docker/docker v26.1.5+incompatible
+	github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.53.1
+)
```

Comment:
This helps compile the collector successfully, but tests with `SPLUNK_ACCESS_TOKEN=<int>` are still failing.

Reply:
Should be resolve by https://github.com/signalfx/splunk-otel-collector/pull/5263/commits/22d38820fa8a44f3652e9032426367d33ae6e221